pkg/apis/certmanager/v1alpha3/types_certificaterequest.go

```diff
@@ -33,6 +33,11 @@ const (
 	// Issued indicates that a CertificateRequest has been completed, and that
 	// the `status.certificate` field is set.
 	CertificateRequestReasonIssued = "Issued"
+
+	// RequestDenied is a Ready condition reason that indicates that a
+	// CertificateRequest has been denied, and the CertificateRequest will never
+	// be issued.
+	CertificateRequestReasonDenied = "RequestDenied"
```

Comment:
Opening for discussion - `RequestDenied` vs `Denied`? Given this is a condition on CertificateRequests, the `Request` part here feels a bit redundant. We don't have `RequestIssued` for example. IDK what's best 🙃

Reply:
Dropping the "Request" makes sense to me